phoebe/frontend/bundle.py

```diff
@@ -10942,6 +10944,8 @@ def compute_pblums(self, compute=None, model=None, pblum=True, pblum_abs=False,
             # this is an internal output used by run_compute, generally not requested by the user
             if system is not None:
                 system.reset(force_recompute_instantaneous=True)
+            
+            print(f'{pblums_abs=} {pblums_scale=} {pblums_rel=} {pbfluxes=}')
```

Comment:
can this be moved to a logger?

Reply:
this was a debugging relic, so it'll be removed altogether in the next commit.